localisation/00_zuwc_united_localisation.csv

```diff
@@ -2340,27 +2329,27 @@ azerothian_opinion;[azerothian.GetName] Opinion;;[azerothian.GetName] Meinung;;;
 night_elf_opinion;[night_elf.GetName] Opinion;;[night_elf.GetName] Meinung;;;;;;;;;;;x
 tauren_opinion;[tauren.GetName] Opinion;;[tauren.GetName] Meinung;;;;;;;;;;;x
 blackhand;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
 blackhand_desc;Known as the Destroyer, Blackhand led the Orcish Horde to Azeroth through the Dark Portal in pursuit of new lands to conquer.;;Blackhand, bekannt als der Zerst�rer, f�hrte die Orkhorde durch das dunkle Portal nach Azeroth, um neue L�nder zu erobern.;;;;;;;;;;;x
 durotan;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
 durotan_desc;One of the few orcs who refused the Blood Pact, Durotan left the Horde after crossing the Dark Portal to uphold ancestral traditions and settled in remote lands with his clan.;;Als einer der wenigen Orks, die den Blutpakt ablehnten, verlie� Durotan die Horde, nachdem er das dunkle Portal durchquert hatte, um die Traditionen der Vorfahren aufrechtzuerhalten, und lie� sich mit seinem Stamm in abgelegenen Gebieten nieder.;;;;;;;;;;;x
 grommash;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
-grommash_desc;The redeemer of the orcs, Grommash earned the title by slaying Mannoroth the Pit Lord, bringing an end to the Blood Pact between the annihilan and the orcish race.;;Der Erl�ser der Orks, Grommash, erwarb den Titel, indem er Mannoroth, den Grubenf�rsten, t�tete und damit den Blutpakt zwischen der annihilanischen und der orkischen Rasse beendete.;;;;;;;;;;;x
+grommash_desc;The redeemer of the orcs, Grommash earned the title by slaying the Pit Lord Mannoroth, bringing an end to the Blood Pact between the annihilan and the orcish race.;;Der Erl�ser der Orks, Grommash, erwarb den Titel, indem er Mannoroth, den Grubenf�rsten, t�tete und damit den Blutpakt zwischen der annihilanischen und der orkischen Rasse beendete.;;;;;;;;;;;x
 orcish_shamanism_opinion;[orcish_shamanism.GetName] Opinion;;[orcish_shamanism.GetName] Meinung;;;;;;;;;;;x
 vashj;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
-vashj_desc;Once chief among Queen Azshara's handmaidens before the Sundering, later a powerful sea witch matron of the Coilfang tribe held in high regard by various Naga.;;Einst die Erste unter den M�gden der K�nigin Azshara vor der Spaltung, sp�ter eine m�chtige Seehexe des Coilfang-Stammes, die von diversen Naga hoch gesch�tzt wurde.;;;;;;;;;;;x
+vashj_desc;Once chief among Queen Azshara's handmaidens before the sundering, later a powerful sea witch matron of the Coilfang tribe held in high regard by various Naga.;;Einst die Erste unter den M�gden der K�nigin Azshara vor der Spaltung, sp�ter eine m�chtige Seehexe des Coilfang-Stammes, die von diversen Naga hoch gesch�tzt wurde.;;;;;;;;;;;x
```

Comment:
**_Sundering_** is a name.
https://wow.gamepedia.com/Great_Sundering

Reply:
done